src/main/java/uk/gov/gchq/palisade/service/CacheService.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
I thought we were getting rid of the CacheService?

Reply:
The CacheService will be removed however, this work has not been completed yet so it is still required in order to get the example to run successfully.
Once the Cache work has been completed the example will need to be reworked to accommodate this change.